utils/symbol.c

```diff
@@ -1430,3 +1423,71 @@ void print_symtabs(struct symtabs *symtabs)
 		symbol_putname(sym, name);
 	}
 }
+
+static unsigned long kaddr;
```

Comment:
I'd like to have a more descriptive name for this.  How about `kernel_base_addr`?